cmd/daprd/components/pubsub_mqtt.go

```diff
@@ -14,7 +14,7 @@ limitations under the License.
 package components
 
 import (
-	"github.com/dapr/components-contrib/pubsub/mqtt"
+	mqtt "github.com/dapr/components-contrib/pubsub/mqtt3"
```

Comment:
In the `RegisterComponent` line below, please add

`pubsubLoader.DefaultRegistry.RegisterComponent(mqtt.NewMQTTPubSub, "mqtt3", "mqtt")`

This should have been added when the latest contribs were pinned (it was in my PR, must have been removed from the PR that superseded mone)

Reply:
Please check now.

Reply:
Is this just an internal rename or  do we have any change? Specially, breaking change?

Reply:
Just a rename of the component. See: https://github.com/dapr/components-contrib/issues/2359